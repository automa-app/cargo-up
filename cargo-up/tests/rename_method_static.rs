use insta::assert_snapshot;
mod utils;

#[test]
fn test_rename_method_static() {
    let (out, err) = utils::run_upgrader("rename_method_static", "0.3.0", true);
    assert_snapshot!(out);
    assert_snapshot!(err);
}
