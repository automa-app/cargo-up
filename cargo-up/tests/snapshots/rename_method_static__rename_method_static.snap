---
source: tests/rename_method_static.rs
expression: out
---

